var/spack/repos/builtin/packages/soqt/package.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Soqt(CMakePackage):
+    """Old Coin GUI binding for Qt, replaced by Quarter"""
+
+    homepage = "https://github.com/coin3d/soqt/"
+    url = "https://github.com/coin3d/soqt/releases/download/v1.6.3/soqt-1.6.3-src.tar.gz"
+    git = "https://github.com/coin3d/soqt/"
+
+    maintainers("paulgessinger")  # reluctantly
```

Comment:
Thanks! It is not a requirement to add yourself as a maintainer. Plenty of packages have no maintainer. But the burden for maintainership is also not very high.